lesson_05/homework_05.py

```diff
@@ -1,39 +1,104 @@
 small_list = [3, 1, 4, 5, 2, 5, 3]
 big_list = [3, 5, -2, -1, -3, 0, 1, 4, 5, 2]
 # task 1. Знайдіть всі унікальні елементи в списку small_list
+print("\n", "Task 01")
+unique_elements = list(set(small_list))
+print(unique_elements)
 
 # task 2. Знайдіть середнє арифметичне всіх елементів у списку small_list
 
+
+print("\n", "Task 02")
+sum_0 = sum(small_list)
+elements = len(small_list)
+arithmetic_mean = sum_0 / elements
+print("Arithmetic mean:", arithmetic_mean)
+
 # task 3. Перевірте, чи є в списку big_list дублікати
 
+print("\n", "Task 03")
+duplicates = len(big_list) != len(set(big_list))
+if duplicates :
```

Comment:
```suggestion
if len(big_list) != len(set(big_list)):
```
без лишніх змінних )